### ISSUE TITLE
Command line arguments?

### ISSUE THREAD

**Issue by MegaApuTurkUltra** [ajyoon/systemf#3]
Does this have support for accessing additional command line args from the brainfuck program? For example
```bash
$ ./systemf program.bf arg1 arg2 arg3
```

**Comment by ajyoon** [comment]
This is not currently supported. There needs to be a way to signal the interpreter to load the arg data into the initial program tape. I don't see a way to do this without extending the language further or by adding a special switch like `$ ./systemf program.bf --args arg1 arg2 arg3 ...` This switch option wouldn't be too difficult, happy to provide support if you want to work on a PR.

**Comment by kailando** [comment]
Maybe we could do a C-style argc/argv and not use --args?
I mean, it wouldn't be too hard - I'm gonna follow this up with a PR.

**Comment by ajyoon** [comment]
@kailando I don't think we should add this. Generally, I view systemf as a fun curiosity and not something under active development. It's not really meant to be practically useful for anything, so these feature requests are kind of beside the point.

**Comment by kailando** [comment]
Fair enough
